app/src/main/kotlin/com/wire/android/notification/broadcastreceivers/NotificationReplyReceiver.kt

```diff
@@ -63,11 +64,19 @@ class NotificationReplyReceiver : BroadcastReceiver() { // requires zero argumen
             with(coreLogic.getSessionScope(qualifiedUserId)) {
                 // TODO better to move dispatcher logic into UseCase
                 CoroutineScope(coroutineContext + dispatcherProvider.io()).launch {
-                    messages.sendTextMessage(qualifiedConversationId, replyText)
-                        .fold(
-                            { updateNotification(context, conversationId, qualifiedUserId, null) },
-                            { updateNotification(context, conversationId, qualifiedUserId, replyText) }
+                    launch {
+                        messages.sendTextMessage(qualifiedConversationId, replyText)
+                            .fold(
+                                { updateNotification(context, conversationId, qualifiedUserId, null) },
+                                { updateNotification(context, conversationId, qualifiedUserId, replyText) }
+                            )
+                    }
+                    launch {
+                        conversations.updateConversationReadDateUseCase(
```

Comment:
`updateConversationReadDateUseCase` will only send last read event to self clients and not send any read receipt

Reply:
my bad it does send it, it was just not clear